schema.graphql

```diff
@@ -4,28 +4,40 @@ schema {
 }
 
 type AccountAddressCreate {
-  errors: [Error!]! @deprecated(reason: "Use typed errors with error codes. This field will be removed after 2020-07-31.")
+  errors: [Error!]!
```

Comment:
Please do not reformat schema file. Just copy from Core repo or edit needed lines.

Reply:
done